Formula/libparc.rb

```diff
@@ -0,0 +1,23 @@
+class Libparc < Formula
+  desc "This library is part of hICN stack"
+  homepage "https://github.com/FDio/cicn/blob/cframework/master/libparc/README.md"
+  url "https://github.com/FDio/cicn", :using=>:git, :branch=>"cframework/master"
+  version "1.0-65~g7944543~b3"
```

Comment:
Is this really the version upstream uses?

Reply:
> Is this really the version upstream uses?

we use this version on the deb packet on ubuntu.
we want to use the same version, if possible.